tailwind.config.cjs

```diff
@@ -20,12 +20,16 @@ module.exports = {
         darkGrey: "#171c1d",
         darkestGrey: "#2C3137",
         accent: "#FAEACE",
+        osGray: "#11181C",
```

Comment:
```suggestion
        osGrey: "#11181C",
```
The spelling works both ways in English, but the project has opted for using "e" in Grey